ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -7164,6 +7166,14 @@ protected Operator genFileSinkPlan(String dest, QB qb, Operator input)
     return output;
   }
 
+  private void commitInsert(Table table, boolean overwrite) {
```

Comment:
Rather change name of method, e.g., to _insert_ or _createInsertDesc_; in the future, the descriptor might be useful beyond commit.